### PR TITLE
Controllers rspec

### PR DESCRIPTION
**Pull request by diegoyon**
### Controllers rspec
- Added rspec gem
- Created request spec file for controllers.
- Added the following tests for authors and posts controllers
1. if response status was correct
2. if a correct template was rendered
3. if the response body includes correct placeholder text

### CODE REVIEW

File: spec/requests/authors_spec.rb [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+    it 'checks if the correct themplate was rendered' do
+      get '/authors'
+      expect(response).to render_template(:index)
+    end
+
+    it 'checks if the response body includes correct placeholder text' do
+      get '/authors'
+      expect(response.body).to include('HERE IS THE LIST OF AUTHORS')
+    end
   end
```
Comment (thecodechaser) [inline spec/requests/authors_spec.rb]: - [ ] As requested in the project requirements you should implement tests for each action. Please implement the tests for the author show action(view).

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+    it 'checks if the correct themplate was rendered' do
+      get '/authors/9/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'checks if the response body includes correct placeholder text' do
+      get '/authors/9/posts'
+      expect(response.body).to include('HERE IS THE LIST OF POSTS FOR A GIVEN AUTHOR')
+    end
   end
```
Comment (thecodechaser) [inline spec/requests/posts_spec.rb]: - [ ] As requested in the project requirements you should implement tests for each action. Please implement the tests for the post show action.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @diegoyon👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] In the README file, Please add steps to run the tests.

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @diegoyon, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**